scripts/tests/chiptest/test_definition.py

```diff
@@ -158,6 +155,18 @@ def __updateSetUpCode(self):
             raise Exception("Unable to find QR code")
         self.setupCode = qrLine.group(1)
 
+    def __terminateProcess(self):
+        if self.process:
+            self.process.terminate()  # sends SIGTERM
+            try:
+                self.process.wait(10)
+            except subprocess.TimeoutExpired:
+                logging.debug('Subprocess did not terminated on SIGTERM, killing it now')
+                self.process.kill()
+                self.process.wait(10)
```

Comment:
nit, Why are we waiting for 10 here? Can we have comment on why magic value was selected for after killing?

Reply:
Please solve the conversion if ok or tell me what I should change. Thank you